### DISCUSSION TITLE
Nextjs routing problem

### DISCUSSION THREAD

**Discussion by Temkit** [vercel/next.js#20642] [Help]
i have a route to a form which is working perfectly when calling from a certain component I doesn't work when calling from another one using the exact same code to route:

                              router.push(
                                `/${encodeURIComponent(
                                  listparams.group
                                )}/${encodeURIComponent(listparams.listkey)}/${
                                  item.id
                                }`
                              )

**Comment by lachlanjc** [comment]
It's hard to help without more context here. Can you provide more code from this component/explain what's going wrong?
